test/acceptance/integration/__init__.py

```diff
@@ -1 +1,24 @@
-__author__ = 'jon'
+# -*- coding: utf-8 -*-
+"""
+Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+
+This file is part of fiware-orion-pep
+
+fiware-orion-pep is free software: you can redistribute it and/or
+modify it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the License,
+or (at your option) any later version.
+
+fiware-orion-pep is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public
+License along with fiware-orion-pep.
+If not, seehttp://www.gnu.org/licenses/.
```

Comment:
`seehttp://www.gnu.org/licenses/ -> see http://www.gnu.org/licenses/`

Maybe you should do an overall review of this... It may happen in old files not included in this PR.